### PR TITLE
checkbox fixes

### PR DESCRIPTION
**Pull request by flexsurfer**
checkbox fixes

### CODE REVIEW

File: src/status_im2/common/contact_list_item/view.cljs [hunk 1]
```diff
@@ -35,6 +35,7 @@
        [quo/icon :i/options {:size 20 :color (colors/theme-colors colors/neutral-50 colors/neutral-40)}]
        [quo/checkbox
         {:default-checked?    @checked?
+         :checked?    @checked?
```
Comment (ibrkhalil) [inline src/status_im2/common/contact_list_item/view.cljs:38]: align map plz

File: src/status_im2/contexts/chat/messages/content/album/view.cljs [hunk 2]
```diff
@@ -6,8 +6,8 @@
             [status-im2.constants :as constants]
             [utils.re-frame :as rf]
             [status-im2.contexts.chat.messages.content.image.view :as image]))
-(def max-display-count 6)
 
+(def max-display-count 6)
```
Comment (ibrkhalil) [inline src/status_im2/contexts/chat/messages/content/album/view.cljs:10]: related?

File: src/status_im2/contexts/chat/messages/content/album/view.cljs [hunk 3]
```diff
@@ -90,8 +90,8 @@
                   :size   :heading-2
                   :style  {:color colors/white}}
                  (str "+" (- images-count (dec max-display-count)))]])]))
-        (:album message))
-      ]
+        (:album message))]
```
Comment (ibrkhalil) [inline src/status_im2/contexts/chat/messages/content/album/view.cljs:93]: related?

**Review by ibrkhalil — APPROVED** [review]

File: src/quo2/components/selectors/selectors.cljs [hunk 4]
```diff
@@ -71,44 +71,47 @@
...
+            :accessibility-role  :checkbox
+            :testID              "checkbox-component"}
+           (when checked?
+             [rn/view
+              {:style
+               {:height 20
+                :width  20}}
+              [icons/icon :i/check-small
+               {:size  20
+                :color colors/white}]])]]]))))
```
Comment (J-Son89) [inline src/quo2/components/selectors/selectors.cljs:115]: does your work fix the issue @OmarBasem has linked in the TODO below? 🤔 
https://github.com/status-im/status-mobile/issues/14681

**Review by J-Son89 — APPROVED** [review]
curious, what was the issue. Should we add a component test to check this behaviour going forward? 🤔

**Pull request closed without merging**